google/cloud/security/common/data_access/forseti.py

```diff
@@ -213,13 +303,71 @@ def _get_latest_snapshot(self):
     def __iter__(self):
         """Main interface to get the data, returns assets and then policies."""
 
-        organization, tables, policies = \
+        organization, folders, tables, policies, group_membership = \
             create_table_names(self.snapshot.cycle_timestamp)
-        yield "organizations", self.session.query(organization).one()
+
+        forseti_org = self.session.query(organization).one()
+        yield "organizations", forseti_org
+
+        # Folders
+        folder_set = (
+            self.session.query(folders)
+            .filter(folders.parent_type == 'organization')
+            .all())
+
+        while folder_set:
+            for folder in folder_set:
+                yield 'folders', folder
+
+            folder_set = (
+                self.session.query(folders)
+                .filter(folders.parent_type == 'folder')
+                .filter(folders.parent_id.in_(
+                    [f.folder_id for f in folder_set]))
+                .all()
+                )
+
         for res_type, table in tables:
-            for item in self.session.query(table).all():
+            for item in self.session.query(table).yield_per(1024):
```

Comment:
You're using 1024 as a 'magic number' here and other places.. Consider making this a global or a kwarg.